dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumResourceScope.kt

```diff
@@ -211,6 +214,14 @@ internal class RumResourceScope(
         sent = true
     }
 
+    private fun resolveResourceDuration(eventTime: Time): Long {
+        val duration = eventTime.nanoTime - startedNanos
+        if (duration <= 0) {
+            devLogger.w(NEGATIVE_DURATION_WARNING_MESSAGE.format(Locale.US, url))
+        }
+        return max(duration, 1)
```

Comment:
you're doing the if check twice here, you could simplify this by doing 
```suggestion
        if (duration <= 0) {
            devLogger.w(NEGATIVE_DURATION_WARNING_MESSAGE.format(Locale.US, url))
            return 1
        } else {
            return duration
        }
```